Index: GD.php
===================================================================
RCS file: /repository/pear/Image_Transform/Driver/GD.php,v
retrieving revision 1.18
diff -u -u -r1.18 GD.php
--- GD.php	29 Dec 2003 18:25:28 -0000	1.18
+++ GD.php	19 Jan 2004 16:55:16 -0000
@@ -228,7 +228,12 @@
     {
         $type           = $type==''? $this->type : $type;
         $functionName   = 'image' . $type;
-        $functionName($this->imageHandle, $filename) ;
+        if ($type == 'jpeg') {
+            $functionName($this->imageHandle, $filename, $quality);
+        }
+        else {
+            $functionName($this->imageHandle, $filename) ;
+        }
         if (!$this->keep_settings_on_save) {
             $this->free();
         }
